utils/clean.sh

```diff
@@ -13,11 +13,30 @@ MYPATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 # ROOT is the root directory of our project.
 ROOT="$MYPATH/.."
 
-pushd "$ROOT"
+pushd "$ROOT" >/dev/null 2>&1
+
+CLEAN_ALL=false
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -a|--all)
```

Comment:
We could probably make an argument for not having an option at all and just deleting all `node_modules` folder always. 

What say you?

Reply:
Tbh I'm personally fine with nuking it all the time! It's fairly quick to install and I find I need to clear out node_modules often enough that I might as well do it all the time. :)

Reply:
I'm going to remove the `--all` flag and just always do it.  After all, if you're running this, it's typically understood that "clean" means... delete everything that isn't in git.